nservicebus/upgrades/5to6.md

```diff
@@ -120,4 +120,11 @@ If you have been using `ISpecifyMessageHandlerOrdering` in a separate assembly s
 
 #### Specifying multiple to run ordered
 
-<!-- import HandlerOrderingWithMultiple -->
\ No newline at end of file
+<!-- import HandlerOrderingWithMultiple -->
+
+## Accessing the builder outside the NServiceBus infrastructure
+When you wanted to use the `IBuilder` interface outside the infrastructure of NServiceBus it was possible to use a hack by casting the `IBus` interface to `UnicastBus` and then accessing the `Builder` property like this:
+
+`var builder = ((UnicastBus)bus).Builder`
+
+This is no longer supported. We advice you to use your container of choice directly without going over the `IBuilder` interface.
```

Comment:
I'm learning, http://blog.oxforddictionaries.com/2014/05/advise-advice/

Ok to pull in?

On Mon, Jul 27, 2015 at 10:25 PM, Daniel Marbach <
daniel.marbach@particular.net> wrote:

> I need grammerly for markdownpad :(
>
>
> Am Montag, 27. Juli 2015 schrieb Simon Cropp :
>
>> In nservicebus/upgrades/5to6.md
>> <https://github.com/Particular/docs.particular.net/pull/716#discussion_r35569796>
>> :
>>
>> > \ No newline at end of file
>> > +<!-- import HandlerOrderingWithMultiple -->
>> > +
>> > +## Accessing the builder outside the NServiceBus infrastructure
>> > +When you wanted to use the `IBuilder` interface outside the infrastructure of NServiceBus it was possible to use a hack by casting the `IBus` interface to `UnicastBus` and then accessing the `Builder` property like this:
>> > +
>> > +`var builder = ((UnicastBus)bus).Builder`
>> > +
>> > +This is no longer supported. We advice you to use your container of choice directly without going over the `IBuilder` interface.
>>
>> We advice you?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Particular/docs.particular.net/pull/716/files#r35569796>
>> .
>>
>